src/Pages/Methods.js

```diff
@@ -4,110 +4,155 @@ const Methods = () => {
   return (
     <div className="page-container">
       <h1>Methodology & Results</h1>
-      <h2>Data Collection</h2>
+      <h2>Data overview</h2>
       <p>
-        The real-time data is scraped from the Chicago Transit Authority's (CTA)
-        bus tracker API's{" "}
+        At a high level, this project compares real-time data (data that indicates which bus trips 
+        actually happened) with schedule data (the trips that the CTA said in advance would happen). 
+        Currently, the information shown on the website reflects data collected from May 20 to November 5, 2022.
+      </p>
+
+      <p>
+        The real-time data is scraped from the CTA bus tracker API 
+        {" "}
         <a
           href="https://www.transitchicago.com/developers/bustracker/"
           target="_blank"
           rel="noreferrer"
         >
           getvehicles feed
-        </a>{" "}
-        every five minutes. Data collection started on 20 May 2022 and continues
-        to date. The scraped data is returned as JSON files, of which there may
-        be hundreds per day, and stored in Amazon's Simple Storage Service (S3).
+        </a>
+        {" "} every five minutes. Data collection started on 20 May 2022 and continues to date, though the data 
+        on this site is not yet automatically updated. Every day we scrape the raw data in the form of hundreds JSON files, 
+        which we store in Amazon Web Service’s Simple Storage Service (S3). We are only using the getvehicles 
+        feed, and not the feed that predicts bus arrivals at specific stops or any of the{" "}
+        <a 
+          href="https://www.transitchicago.com/assets/1/6/cta_Bus_Tracker_API_Developer_Guide_and_Documentation_20160929.pdf"
+          target="_blank"
+          rel="noreferrer">
+          other bus data feeds that the CTA makes available.
+        </a>
+      
       </p>
       <p>
-        Schedule data is collected from the CTA General Transit Feed
-        Specification (GTFS) feed for recent feeds and from{" "}
+        Schedule data is collected from the CTA General Transit Feed Specification (GTFS) feed data. 
+        A list of schedule versions starting from the beginning of data collection on 20 May 2022 up to 
+        the current date is compiled from{" "}
```

Comment:
Should we add "up to November 3, 2022" instead of "current date" here? This would correspond to the sentence above i.e. "Currently, the information shown on the website reflects data collected from May 20 to November 5, 2022."

Reply:
Just fixed thank you for catching!